lib/db2.js

```diff
@@ -76,6 +76,8 @@ DB2.prototype.create = function(model, data, options, callback) {
  * @param {Function} cb The callback function
  */
 DB2.prototype.update = function(model, where, data, options, cb) {
+  debug('DB2.prototype.update (enter): where=%j, data=%j, options=%j\n',
```

Comment:
While in this example there is no exit it is nice to see enter vs. exit for other functions and sticking to a standard is helpful.

Commit descriptions are 50 characters max and I'm not sure they are useful.  This is pretty self explanatory.